cmd/flags.go

```diff
@@ -70,6 +70,14 @@ func addOutputFormat(fs *pflag.FlagSet, outputFormat *util.EnumFlag) {
 	fs.VarP(outputFormat, "format", "f", "set output format")
 }
 
+func addListAnnotations(fs *pflag.FlagSet, value *bool) {
+	fs.BoolVarP(value, "annotations", "a", false, "list annotations")
+}
+
+func addAnnotationsFilter(fs *pflag.FlagSet, paths *[]string) {
```

Comment:
I'm wondering if a filtered path should be recursively applied, so that all child packages and -rules are also included; or if that should be handled by a wildcard. If `*` is used for a wildcard, then we'd need to encapsulate the path in `"` for it to not be caught as an expansion by the terminal; e.g. `opa inspect -a -p "data.foo.*"`

Reply:
> If * is used for a wildcard, then we'd need to encapsulate the path in " for it to not be caught as an expansion by the terminal; e.g. opa inspect -a -p "data.foo.*"

That's unavoidable, I think, and something we could live with. WDYT?

OTOH there's a certain expectation attached to globs. You could think that `data.foo.*` does not match `data.foo.bar.baz`, but that only `data.foo.**` would, or something... we have no such convention for globbing refs in place yet, so we should introduce that with care... 🤔 

Perhaps we could have another flag for _"all annotations under this prefix"_? I don't know what's better just like that...